Release Cognitive Services Data Plane SDKs Round I (#4955)
Azure/azure-sdk-for-js: sdk/cognitiveservices/cognitiveservices-personalizer/README.md

#### nodejs - Authentication, client creation and reward events as an example written in TypeScript.

##### Install @azure/ms-rest-azure-js

```bash
npm install @azure/ms-rest-azure-js
```

##### Sample code

> The following sample lets you rank a personalized request object. To know more, Refer the [Azure Documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/personalizer/)

Reviewer: ```suggestion
The following sample ranks a personalized request object. To know more, refer to the [Azure Documentation on Personalizer](https://docs.microsoft.com/en-us/azure/cognitive-services/personalizer/)
```
